cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -3861,4 +3861,12 @@
     </description>
   </property>
 
+  <property>
+    <name>profile.creation.update.allowed</name>
```

Comment:
this is kind of an awkward name, let's just name it profile.updates.allowed.

Reply:
Done. Removed the "s".